src/cluster/placement/types.go

```diff
@@ -440,14 +440,14 @@ const (
 // Storage provides read and write access to placement.
 type Storage interface {
 	// Set writes a placement.
-	Set(p Placement) error
+	Set(p Placement) (int, error)
```

Comment:
Hey, see how hard this is but I think either of these two approaches is a good idea:

1)
So, we can either update the description for these methods to be:
```go
// Set writes a placement and returns the version or an error.
Set(p Placement) (int, error)
```

2)
Or we can:
```go
package kv

// Version is the version of a key value pair.
type Version int
```

```go
package placement

type Storage interface {
  // Set writes a placement.
  Set(p Placement) (kv.Version, error)

  // ...
}
```

Reply:
(If it's not clear, my preference is number two, but yeah I could see how it might be intimidating to add this change - having said that if you scope the change super tight then it shouldn't be too full on to make the change)

Reply:
Or you could do
```
Set(p Placement) (Placement, error)
```
so the users can do 
```
p, err := storage.Set(p)
if err != nil {...}
v := p.Version()
```